src/Workspaces/CSharp/Portable/CodeGeneration/CSharpCodeGenerationService.cs

```diff
@@ -113,6 +113,11 @@ protected override TDeclarationNode AddMethod<TDeclarationNode>(TDeclarationNode
         {
             CheckDeclarationNode<TypeDeclarationSyntax, CompilationUnitSyntax, NamespaceDeclarationSyntax>(destination);
 
+            if (options.Options == null)
+            {
+                options = options.With(options: Workspace.Options);
```

Comment:
`options.Options ??= ...`

Reply:
Thanks so much for the review. If I'm not mistaken, since options.Options doesn't have a set method, I don't think we can assign to it. We could have options.Options support a set method, but I don't know if this is the right move since none of the existing settings in CodeGenerationOptions have set methods.

Reply:
sorry...  sould be:  `options = options.With(options: options.options ?? Workspace.Options)`

Reply:
Thanks! Updated.